prow/config_knative.yaml

```diff
@@ -670,12 +670,12 @@ periodics:
       secret: s390x-cluster1
   - custom-job: s390x-kourier-tests
     release: "1.0"
-    cron: 0 15 * * *
+    cron: 20 3 * * *
```

Comment:
Just out of curiosity, how did you determine these crons?

Reply:
Thanks for the question. The CI is not a big scale atm. I set the schedule just on my own (just simple math based on the job running time and an amount of jobs). :wink:

Reply:
At this time, another 45 sets of custom jobs will be put in place. So I need to pack the current schedules as much as possible to accommodate them.